critiquebrainz/frontend/templates/review/modify/artist.html

```diff
@@ -1,11 +1,11 @@
 {% if entity is not defined %}
-  {% set entity = review.entity_id | entity_details(type=entity_type) %}
```

Comment:
I'm a little bit unsure why we have this check at the beginning of all of these .html files (the `if entity is not defined` check). It seems like these files are imported [from here](https://github.com/metabrainz/critiquebrainz/blob/a60a4731e1ff28327b0ff4ccf871cbe3b8f83d9a/critiquebrainz/frontend/templates/review/modify/base.html#L14), and this file is rendered from `frontend/views/review.py`, and the `entity` context variable is added here.

I recommend that we remove these 3 lines from these template files. Further below in the file when you need the entity_id, can we just use `entity.id`